source/_patterns/01-elements/loading-indicator/loading-indicator.hbs

```diff
@@ -0,0 +1,16 @@
+<svg
+	class="elm-loadingindicator"
+	viewBox="0 0 44 44"
+	aria-labelledby="elm-loadingindicator__description" role="img"
+	{{#if size }} data-size="{{ size }}"{{/if }}
+>
+	<title id="elm-loadingindicator__description">{{aria-description}}</title>
+	<circle
+		class="elm-loadingindicator__circle"
+		cx="22"
+		cy="22"
+		r="20"
+		fill="none"
+		stroke-miterlimit="10"
+	></circle>
```

Comment:
Let's extract this to an external SVG as aligned.

Reply:
@mfranzke don't we have the problem to style external svgs via css in this solution? When loading by href the svg source is not part of the dom or do I oversee something?

Reply:
If you would reference the external image via `<svg><use href /></svg>` (compare to the following code), than its possible to style the SVG parts via CSS:
https://github.com/db-ui/core/blob/de4e73f91127a7748a8a3ea0d5d9eb28cb837e19/source/_patterns/00-base/icons/illustration-icons.md?plain=1#L10-L17

Reply:
@mfranzke as discussed earlier today, I don't see big advantages in using external svgs due to the fact that all animations are done with css and the pure svg won't be used without those css definitions. I'll recommend using the loading spinner as whole component e.g. for using in buttons